src/test/java/hudson/plugins/copyartifact/TriggeredBuildSelectorTest.java

```diff
@@ -24,21 +24,29 @@
 
 package hudson.plugins.copyartifact;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
 
 import java.io.IOException;
 import java.util.Arrays;
 import java.io.File;
-import org.apache.commons.io.FileUtils;
 
-import hudson.Util;
-import hudson.maven.MavenModuleSet;
 import hudson.model.Cause;
+import hudson.model.FreeStyleBuild;
 import hudson.model.FreeStyleProject;
+import hudson.model.ParameterDefinition;
 import hudson.model.ParametersAction;
-import hudson.model.StringParameterValue;
-import hudson.model.FreeStyleBuild;
+import hudson.model.ParametersDefinitionProperty;
 import hudson.model.Result;
+import hudson.model.StringParameterDefinition;
+import hudson.model.StringParameterValue;
+import org.apache.commons.io.FileUtils;
+
+import hudson.Util;
+import hudson.maven.MavenModuleSet;
```

Comment:
> I suppose this sorting of imports is performed by your IDE.
> Could you revert this change?

Because:
* It may cause other pull requests conflict.
* It gets difficult to track changes from git logs.